README.md

```diff
@@ -124,3 +124,39 @@ operator-sdk test local ./test/e2e --up-local --kubeconfig=$KUBECONFIG --namespa
 ```
 
 You can also run the tests with `make e2e-test`.
+
+## Uninstalling an Operator from the CLI
+
+After an operator has been installed from marketplace, to uninstall the operator from a particular OperatorGroup you need to delete the following resources. Below we are using the descheduler operator as an example:
+
+1. Delete the installation `CatalogSourceConfig` in the `openshift-marketplace` that contains the installed operator package. If created via the OperatorHub UI, it will be named `installed-<OPERATORSOURCE>-<TARGETNAMESPACE>` based on the `OperatorSource` that the operator comes from and the `targetNamespace` that the operator was installed to.
```

Comment:
This should be the last step. Start with if there are multiple operators installed.

Please say `OpenShift OperatorHub UI`.